bangazon/bangazon/urls.py

```diff
@@ -13,9 +13,15 @@
     1. Import the include() function: from django.conf.urls import url, include
     2. Add a URL to urlpatterns:  url(r'^blog/', include('blog.urls'))
 """
-from django.conf.urls import url
-from django.contrib import admin
+
```

Comment:
I propose that we leave the admin panel inside for testing purposes
```from django.contrib import admin```

A lot of good stuff here.